docs/release-notes/artifacts/pr0249.yaml

```
@@ -0,0 +1,12 @@
+schema_version: 1
+changes:
+  - title: "Add HTTP/2 support to HAProxy"
```

```suggestion
  - title: Added HTTP/2 support to HAProxy
```
Nitpick
I removed the quotation marks since the other artifacts don't have them, but I think you should be fine either way